lib/playground.dart

```diff
@@ -187,7 +191,12 @@ class Playground {
 
     keys.bind('ctrl-s', _handleSave);
     keys.bind('ctrl-enter', _handleRun);
-    keys.bind('f1', _handleHelp);
+    keys.bind('f1', () => _toggleDocTab());
+    document.onKeyUp.listen((e) => _handleHelp());
+    document.onClick.listen((e) => _handleHelp());
```

Comment:
Sure, makes sense. I just want to avoid sending a request on every keystroke. But doing it for up and down arrow keys when the completion window is open makes sense.